cpp/cmake_modules/FindPythonLibsNew.cmake

```diff
@@ -234,11 +234,7 @@ FUNCTION(PYTHON_ADD_MODULE _NAME )
       SET_TARGET_PROPERTIES(${_NAME} PROPERTIES LINK_FLAGS
                           "-undefined dynamic_lookup")
     ELSE()
```

Comment:
The `-undefined dynamic_lookup` is still being used on OS X but it's not needed for Linux or Windows